### PR TITLE
[13.0][MIG] stock_available_mrp

### PR DESCRIPTION
**Pull request by ps-tubtim**
This PR is waiting for https://github.com/OCA/stock-logistics-warehouse/pull/802, you can always use this for readability.
- [x] [13.0][MIG] stock_available

### CODE REVIEW

**Comment by m-abdalrahman** [comment]
hello @ps-tubtim I can't enter the products screen after installing the module
Screenshot from 2020-06-15 14-18-38

log:
`odoo.http: ('product.product(1,).bom_id', None)
`

after add store=True to bom_id field to fix the previous issue, when I Create new product potential quantities values not calculated, but if I uninstall stock_available and install stock_available_mrp  potential quantities values calculated and appear

**Comment by rousseldenis** [comment]
@ps-tubtim You can rebase as #802 is merged

**Comment by i-vyshnevska** [comment]
@ps-tubtim can you pls rebase and check travis build for errors

**Comment by leemannd** [comment]
Hello @ps-tubtim a fix is included into https://github.com/OCA/stock-logistics-warehouse/pull/1070

File: stock_available_mrp/models/product_product.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    def _compute_bom_id(self):
+        bom_obj = self.env["mrp.bom"]
+        boms = bom_obj.search(self._get_bom_id_domain(), order="sequence, product_id")
+        for product in self:
+            product_boms = boms.filtered(
+                lambda b: b.product_id == product
+                or (not b.product_id and b.product_tmpl_id == product.product_tmpl_id)
+            )
+            if product_boms:
+                product.bom_id = first(product_boms)
```
Comment (leemannd) [inline stock_available_mrp/models/product_product.py:43]:
```suggestion
                product.bom_id = first(product_boms)
            else:
                product.bom_id = False
```

**Review by leemannd — COMMENTED** [review]
Missing returb

**Review by leemannd — CHANGES_REQUESTED** [review]
Missing return in compute function. This will lead to the impossibility to see products

**Comment by chienandalu** [comment]
We just merged this changes due to Odoo upstream changes that intruduced some incompatibilities. https://github.com/OCA/stock-logistics-warehouse/pull/1099/files Please include it in your migration

File: stock_available_mrp/models/product_product.py (hunk 1 above)
Comment (ps-tubtim) [reply stock_available_mrp/models/product_product.py:43]: Thank you for your review. Edited.

**Comment by leemannd** [comment]
This can be closed. The module has been migrated into another PR

**Pull request closed without merging**